pkg/cosign/fulcio/fulcio.go

```diff
@@ -165,7 +153,7 @@ type Signer struct {
 	*signature.ECDSASignerVerifier
 }
 
-func NewSigner(ctx context.Context, idToken string) (*Signer, error) {
+func NewSigner(ctx context.Context, idToken, oidcIssuer string, oidcClientID string, oidcClientSecret string, fulcioClient string, rekorClient string) (*Signer, error) {
```

Comment:
You can combine all these argument names and use one `string` argument type e.g. `oidIssuer, oidcClientID, oidcClientSecret, fulcioClient, rekorClient string` to be more Go idiomatic. This comment applies above as well.

Reply:
Any reason not to keep these named `fulcioURL` and `rekorURL` for consistency?

Reply:
> You can combine all these argument names and use one string argument type e.g. oidIssuer, oidcClientID, oidcClientSecret, fulcioClient, rekorClient string to be more Go idiomatic. This comment applies above as well.

Yup, good call.

> Any reason not to keep these named fulcioURL and rekorURL for consistency?

I think I was sticking to the convention already set in the struct, I will have check.

Reply:
I'd be in favor of changing these to "URL" or "Endpoint" to be more descriptive

Reply:
Looking into it we don't need rekorClient there(so removed it) and the fulcioClient is an connection instance